File: exercicios/R/1050/1050.r
input = file('stdin','r')

a = as.integer(readLines(input,n=1))

if(a == 61){
    write(sprintf("Brasilia"),'')
}else if(a == 71){
    write(sprintf("Salvador"),'')
}else if(a == 11){
    write(sprintf("Sao Paulo"),'')
}else if(a == 21){
    write(sprintf("Rio de Janeiro"),'')
}else if(a == 32){
    write(sprintf("Juiz de Fora"),'')
}else if(a == 19){
    write(sprintf("Campinas"),'')
}else if(a == 27){
    write(sprintf("Vitoria"),'')
}else if(a == 31){
    write(sprintf("Belo Horizonte"),'')
}else{
    write(sprintf("DDD nao cadastrado"),'')
}

File: exercicios/JavaScript/1050/1050.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

var op = {61:"Brasilia", 71:"Salvador", 11:"Sao Paulo", 21:"Rio de Janeiro", 32:"Juiz de Fora", 19:"Campinas", 27:"Vitoria", 31:"Belo Horizonte"}

var cod = lines.shift()

if(op[cod]){
    console.log(op[cod])
}else{
    console.log("DDD nao cadastrado")
}